core/src/test/scala/sttp/tapir/CodecTest.scala

```diff
@@ -73,13 +74,7 @@ class CodecTest extends AnyFlatSpec with Matchers with Checkers {
     checkEncodeDecodeToString[Uri]
     checkEncodeDecodeToString[BigDecimal]
     checkEncodeDecodeToString[JBigDecimal]
-    checkEncodeDecodeToString[LocalTime]
-    checkEncodeDecodeToString[LocalDate]
-    checkEncodeDecodeToString[OffsetDateTime]
-    checkEncodeDecodeToString[Instant]
-    checkEncodeDecodeToString[ZoneOffset]
```

Comment:
what about tests for these types? would be good if they are still covered

Reply:
Added missing tests